http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -473,6 +485,36 @@ public HttpServerConfig setCompressionEnabled(boolean compressionEnabled)
         return this;
     }
 
+    @MinDataSize("128MB")
+    @MaxDataSize("32GB")
```

Comment:
this limit seems artificial

Reply:
the default would reach this value if the heap size is 245 GB

Reply:
Do you suggest 256GB should be enough for everybody?

Reply:
Do we even need a max allowed value here?